default.nix

```
@@ -46,6 +58,7 @@ python3Packages.buildPythonApplication {
         [
           pkgs.nixVersions.stable or nix_2_4
```

```suggestion
          pkgs.nixVersions.stable
```
I don't think that is necessary anymore